Makefile

```diff
@@ -6,28 +6,20 @@ include ./Makeconf
 .PHONY: all install uninstall
 all install uninstall: lisp doc etc
 
-.PHONY: version
```

Comment:
No need to remove the entire version, just the git within it. The most useful part of that vesion is the Emacs version which people are often confused about.

Reply:
OK, done